### PR TITLE
Dashboard Table cleanups

### LINKED ISSUE #1783: MaterialUI followup: Tables
The dashboard was converted to material-ui in #1776.

Tables throughout the app need to be revisited and neatened
  - [ ] fix table widths, currently the tables are too wide, the columns are too narrow
 - [ ] the tables in Overview are currently overflowing the Expand container
- [ ] now that #1225 has merged, remove the restriction on displaying the grafana link in the tables for authorities
- [ ] the bar chart in the ServiceMesh table should have a greyed out bar for no meshed, not yellow

### PR DESCRIPTION
**Pull request by rmars**
This branch: 
- Removes the restrictions on the MetricsTables for Authorities Grafana links (Authorities Grafana dashboards were added in #1772)
- Fixes the tables overflowing their containers on the Overview page
- Allows tables to be denser, allowing for more data on screen
- Fixes the colour of the meshed status bar in the ServiceMesh page
- fixes the ErrorModal icon alignment and colour
- small appearance tweaks to the things in the table e.g. icons

## Before
screen shot 2018-10-19 at 5 49 29 pm

## After
screen shot 2018-10-19 at 5 49 09 pm

## Before
screen shot 2018-10-19 at 5 35 09 pm

## After
screen shot 2018-10-19 at 5 34 48 pm

Fixes #1783

### CODE REVIEW

File: web/app/js/components/ErrorModal.jsx [hunk 1]
```diff
@@ -145,15 +145,17 @@ class ErrorModal extends React.Component {
         <Tooltip title="Pods are initializing"><CircularProgress size={20} thickness={4} /></Tooltip>
       );
     } else {
-      return <ErrorIcon onClick={this.handleClickOpen} />;
+      return (
+        <ErrorIcon color="error" fontSize="small" onClick={this.handleClickOpen} />
```
Comment (klingerf) [inline web/app/js/components/ErrorModal.jsx]: Similar to my other comment -- would it be possible to make the height of this icon match the height of the text here, and remove the padding on the bottom?

image

File: web/app/js/components/NamespaceLanding.jsx [hunk 2]
```diff
@@ -17,7 +18,7 @@ import { withContext } from './util/AppContext.jsx';
 
 const isMeshedTooltip = (
   <Tooltip title="Namespace is meshed" placement="right-start">
-    <CheckCircleIcon />
+    <CheckCircleIcon color="primary" />
```
[the remark this answers is not in the document]
Comment (rmars) [reply web/app/js/components/NamespaceLanding.jsx]: that is sooooo strange! the icons show up for me on Chrome and Safari, but not Firefox

File: web/app/js/components/ErrorModal.jsx (hunk 1 above)
Comment (rmars) [reply web/app/js/components/ErrorModal.jsx]: That's so strange... it's aligned for me:
screen shot 2018-10-24 at 11 00 50 am

File: web/app/js/components/NamespaceLanding.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (rmars) [reply web/app/js/components/NamespaceLanding.jsx]: I've made https://github.com/linkerd/linkerd2/issues/1805 to track this, which I'll address soon after merging this branch!

File: web/app/js/components/ErrorModal.jsx (hunk 1 above)
Comment (rmars) [reply web/app/js/components/ErrorModal.jsx]: I figured it out, I was looking at the icons in the StatusTable, but the ones on the MetricsTable aren't aligned!

Comment (klingerf) [reply web/app/js/components/ErrorModal.jsx]: Ahhh, that makes sense.

**Pull request merged**